src/audacious-discord-rpc.cpp

```diff
@@ -40,20 +40,30 @@ const char RPCPlugin::about[]
       "Displays the current playing track as your Discord status.\n"
       "(Discord should be running before this plugin is loaded.)";
 
+static const ComboItem statusDisplayItems[]
```

Comment:
Prefer snake\_case for this project. While different dependencies use different standards, Audacious (and transitively this project) uses snake case for variables.